src/App.css

```diff
@@ -1,39 +1,60 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 .App {
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.Calculator {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: repeat(4, 60px);
+  grid-template-rows: repeat(6, 60px);
+  height: 100vh;
```

Comment:
- [ ] Kindly fix these borders in order to match the design better 
![image](https://user-images.githubusercontent.com/62705947/221053864-97177b96-f09d-4280-9aa0-e8d123e0b276.png)